stock_inventory_valuation_report/security/ir.model.access.csv

```diff
@@ -0,0 +1,4 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_report_stock_inventory_valuation_report,report.stock.inventory.valuation.report,model_report_stock_inventory_valuation_report,base.group_user,1,1,1,1
+access_stock_inventory_valuation_view,stock_inventory_valuation_view,model_stock_inventory_valuation_view,base.group_user,1,1,1,1
+access_report_s_i_v_r_report_stock_inventory_valuation_report_xlsx,report.s_i_v_r.report_stock_inventory_valuation_report_xlsx,model_report_s_i_v_r_report_stock_inventory_valuation_report_xlsx,base.group_user,1,1,1,1
```

Comment:
**issue**: IMO `base.group_user` is too broad. At the very least, we should restrict access to `stock.group_stock_user`

Reply:
thanks, I'll fix it